examples/app/stock_applet/stock_data.py

```diff
@@ -0,0 +1,49 @@
+import os
+import urllib
+import zipfile
+
+
+def download_data(url, save_dir, exclude_term):
+    name = os.path.join(save_dir, 'temp.zip')
+
+    #get the zip file
+    try:
+        name, hdrs = urllib.urlretrieve(url, name)
```

Comment:
I am pretty sure `urllib.urlretrieve` is python2 specific... you can probably use `six` to make also python3 compatible... let me see... ok, look at this: https://pythonhosted.org/six/#module-six.moves.urllib.request